### DISCUSSION TITLE
Clear data in PVC

### DISCUSSION THREAD

**Discussion by northboys** [longhorn/longhorn#3713] [Q&A]
I am using Longhorn for kubernetes persistence. I have pvc which is full, how can I clear all data in pvc while keeping pvc resource?

**Comment by jenting** [comment]
There are two ways:
1. Delete PVC and recreate.
2. Execute to the Pod, go to the mount point and clear all the data https://www.reddit.com/r/kubernetes/comments/igvetc/delete_pvc_data/
